File: mf.py
l = [1,1,2,2,2,3,3,4,4,4,4,4,4]

cnt = 0
best = 0
for i in range(len(l) - 1):
    if l[i] == l[i + 1]:
        cnt += 1
    else:
        best = max(cnt, best)
        cnt = 0

print(best)
    

File: mf.cpp
#include <iostream>
#include <bits/stdc++.h>
using namespace std;

int main() {
	int size;
	cin >> size;
	vector<int> v;
	for (int i = 0; i < size; i++) {
		int n;
		cin >> n;
		v.push_back(n);
	}

	sort(v.begin(), v.end());
	cout << "sorted" << '\n';
	int sum = 0;
	int best = 0;
	
	for (int j = 0; j < v.size() - 1; j++) {
		sum = max(j, sum + j);
		best = max(best, sum);
	}
	
	cout << best << '\n';
	return 0;
}
